test/transpile/plugins/transformer-plugin.js

```diff
@@ -0,0 +1,18 @@
+/**
```

Comment:
Where is the `.ts` source for this file?

Reply:
I did not add TS source as it would mean adding another step to our test or test run to compile it. We could do that at some point if we have more code like this, but for now I don't know if it would add much.